src/components/LoginForm.tsx

```diff
@@ -28,12 +28,26 @@ const LoginForm = () => {
     );
   };
 
+  const helpSteps = [
```

Comment:
this looks like a constant as well 
could we move those together?

Reply:
Done.